usaspending_api/api_contracts/contracts/v2/awards/award_id.md

```diff
@@ -218,47 +218,47 @@ This endpoint returns a list of data that is associated with the award profile p
 
 ## Recipient (object)
 + `recipient_name`: `Booz Allen Hamilton` (required, string, nullable)
-+ `recipient_hash`: `e4096343-5e8f-352a-f8af-d2a8b0f0ae68-C` (required, string)
++ `recipient_hash`: `e4096343-5e8f-352a-f8af-d2a8b0f0ae68-C` (required, string, nullable)
 + `recipient_unique_id`: `2424224` (required, string, nullable)
     The recipient's DUNS
 + `parent_recipient_name`: `HoneyWell` (string, nullable)
 + `parent_recipient_hash`: `18e9854a-6e51-29fe-0add-4f2ad80a4010-P` (string, nullable)
 + `parent_recipient_unique_id`: `2424232` (required, string, nullable)
     The recipient's parent's DUNS
 + `location` (required, Location, nullable)
-    The recipeint's location
+    The recipient's location
 + `business_categories` (required, array[string])
-    Names of the recipients' business catagories in human readable format
+    Names of the recipients' business categories in human readable format
 
-## PeriodOfPerformance
+## PeriodOfPerformance (object)
 + `start_date`: `2004-02-19` (required, string)
 + `end_date`: `2005-02-19` (required, string)
     Corresponds to database fields current_end_date for contracts and ordering_period_end_date for IDVs
 + `last_modified_date`: `2301-02-20` (required, string)
 + `potential_end_date`: `2301-02-23` (required, string, nullable)
 
-## PeriodOfPerformanceAssistance
+## PeriodOfPerformanceAssistance (object)
 + `start_date`: `2004-02-19` (required, string, nullable)
 + `end_date`: `2005-02-19` (required, string, nullable)
 + `last_modified_date`: `2301-02-20` (required, string, nullable)
 
-## Location
+## Location (object)
 + `address_line1`: `123 Sesame St` (required, string, nullable)
 + `address_line2` (required, string, nullable)
 + `address_line3` (required, string, nullable)
 + `foreign_province` (required, string, nullable)
 + `city_name`: `McLean` (required, string, nullable)
 + `county_name`: `Fairfax` (required, string, nullable)
 + `state_code`: `VA` (required, string, nullable)
-+ `state_name`: `Virginia` (required, string, nullable)
++ `state_name`: `Virginia` (optional, string, nullable)
```

Comment:
`state_name` is optional but all the other fields are required?

Reply:
When the location is used as part of the recipient it doesn't include `state_name`. To be more accurate I could create a copy of the Location object that doesn't include `state_name` for the recipient and does in other instances.

Reply:
Makes sense. I think that was an oversight on the ticket to add state name to the response.

Leave the contract as-is, I'll make a note to address this in the future